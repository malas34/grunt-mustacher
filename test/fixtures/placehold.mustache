<html>
    <head>
        <title>{{title}}</title>
    </head>
    <body>
        <h3>Dummy</h3>
        <p>{{$dummy '50'}}</p>
        <p>{{$dummy}}</p>
        <p>{{$dummy '320x100'}}</p>
        <p>{{$dummy '320x100:placehold+image'}}</p>
        <p>{{$dummy '320x100:ACE539:placehold+image'}}</p>
        <p>{{$dummy '320x100:FF99EE:FFF:placehold+image'}}</p>
    </body>
</html>
