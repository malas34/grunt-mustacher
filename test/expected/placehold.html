<html>
    <head>
        <title>Titre de la page</title>
    </head>
    <body>
        <h3>Dummy</h3>
        <p><img src="http://placehold.it/50" alt="" title="" /></p>
        <p><img src="http://placehold.it/300" alt="" title="" /></p>
        <p><img src="http://placehold.it/320x100" alt="" title="" /></p>
        <p><img src="http://placehold.it/320x100&text=placehold+image" alt="" title="" /></p>
        <p><img src="http://placehold.it/320x100/ACE539&text=placehold+image" alt="" title="" /></p>
        <p><img src="http://placehold.it/320x100/FF99EE/FFF&text=placehold+image" alt="" title="" /></p>
    </body>
</html>
